src/test/kotlin/com/igorwojda/list/capitalizeFirst/challenge.kt

```diff
@@ -4,7 +4,7 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun capitalizeFirst(list: List<String>): List<String> {
-    TODO("not implemented")
+    return if (list.isEmpty()) list else listOf(list[0].capitalize()) + capitalizeFirst(list.drop(1))
```

Comment:
coding suggestion: always expand expression and use paranthesis accordingly:
```
val result = if (condition) {
  value1
} else {
  value2
}
```

It makes your code much more legible + you don't get into trouble with compound expressions